### DISCUSSION TITLE
Would you like to contribute to the book?

### DISCUSSION THREAD

**Discussion by thomasfermi** [thomasfermi/Algorithms-for-Automated-Driving#4] [Ideas]
Hi everyone 👋

Are you interested in contributing to the book by adding a new chapter? Or do you have other ideas for improvements? If this is the case, please let me know here, so we can coordinate 😃I would love to hear your ideas!

**Comment by thomasfermi** [comment]
I created an issue for a chapter on camera calibration: #3  

Please check it out 😉

**Comment by thomasfermi** [comment]
I created an issue for a chapter on Model Predictive Control : #5 

Please check it out 😉

**Comment by Kd-Here** [comment]
Hey Thomas!
I loved your project it's work really well.
I wanted to learn how to contribute to but your project is to difficult for me to help.
Let me know if you need any begineer help on other project would love to help and learn.

**Reply by thomasfermi** [reply]
Hi! Glad to hear that you liked it. Yeah, when I find a smaller thing that can be added/changed I will create a GitHub issue and will let you know :)
